catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -484,4 +497,8 @@ private void deleteImageFiles() {
             }
         });
     }
+
+    public void setMaxFileSizeMB(int maxFileSizeMB) {
+        this.maxFileSizeMB = maxFileSizeMB;
```

Comment:
If we change to storing max size as bytes we can still leave this function as taking in megabytes for ease of use.